Procfile

```diff
@@ -0,0 +1 @@
+web: java -Dserver.port=$PORT $JAVA_OPTS -javaagent:newrelic/newrelic.jar -jar target/play-ping-pong.jar
```

Comment:
@rydogs Should this be ignored? Not sure if this is an eclipse file or something you added.